docs/run-oob-locally.md

```diff
@@ -0,0 +1,150 @@
+```
+Connect to one of your mongod:
+```
+mongo --port 27018
+```
+
+initiate the replica set:
+You can create a replica set configuration object in mongo environment, as in the following example:
+```
+rsconf = { _id: "rs0", members: [ { _id: 0, host: "localhost:27018" }, { _id: 1, host: "localhost:27019" }, { _id: 2, host: "localhost:27020" } ] }
+```
+
+use `rs.initiate()` to initiate the replica set:
+```
+rs.initiate( rsconf )
+```
+
+(SKIP THIS STEP - Only use it to clean up your mongo) Clean up:
+```
+rm -rf /tmp/mongodb/rs0-0/*  /tmp/mongodb/rs0-1/* /tmp/mongodb/rs0-2/*
+```
+
+## CloudServer
+
+/!\ IMPORTANT: locationConfig.json should be identical to the backbeat conf/locationConfig.json.
+Add Ring access key and secret key in locationConfig.json:
+https://github.com/scality/backbeat/blob/1657782020009d63be8f3df342917305dea7671b/conf/locationConfig.json
+
+```
+REMOTE_MANAGEMENT_DISABLE=true S3VAULT=multiple yarn start_mongo
+```
+
+## Vault
+```
+VAULT_DB_BACKEND="MONGODB" yarn start
+```
+
+## Redis
+
+Note: Redis is used for its “publish/subscribe messages” feature. 
+Client sends an authenticated request (ex. resume ingestion for location1) to CloudServer that gets proxied to Backbeat API server. 
+Backbeat API will publish messages to “backbeat-ingestion-location1” channel with a “resume” state. 
+Ingestion producer subscribes to “backbeat-ingestion-*”, receives messages and updates application and zookeeper state.
+```
+redis-server /usr/local/etc/redis.conf
+```
+
+## Backbeat - Backbeat API
+```
+git checkout feature/ARTESCA-1849/poc
+```
+Add Ring access key and secret key in conf/locationConfig.json:
+```
+"accessKey": "TO_BE_ADDED",
+"secretKey": "TO_BE_ADDED"
+```
+Start backbeat API:
+```
+MANAGEMENT_BACKEND=operator REMOTE_MANAGEMENT_DISABLE=true yarn run start
+```
+
+## Backbeat - Ingestion producer
+```
+MANAGEMENT_BACKEND=operator yarn ingestion_populator
+```
+## Backbeat - Ingestion processor
+```
+MANAGEMENT_BACKEND=operator yarn run mongo_queue_processor
+```
+## VaultClient
+Create an IAM account (account1) and its keys:
+```
+bin/vaultclient create-account --name account1 --email dev@null1 --port 8600
+bin/vaultclient generate-account-access-key --name account1 --port 8600
+aws configure --profile account1
+```
+## AWS S3 CLI
+
+Create a bucket into this account with ingestion enabled location:
+```
+aws s3api create-bucket --bucket bucket1 --create-bucket-configuration LocationConstraint=wontwork-location:ingest --endpoint-url http://127.0.0.1:8000 --profile account1
+```
+
+Check the source bucket exists and fetch its objects.
+```
+aws s3api list-objects --endpoint-url http://217.182.11.36 --profile s3consoleaccount --bucket ring-bucket-1
+```
+
+## Zenko-client
+
+Resume ingestion for location “wontwork-location”
+Note: by default locations ingestion state is set to paused.
+```
+git checkout feature/ARTESCA-1849/poc
+node oob.js
```

Comment:
Where is this file? I don't see it anywhere.

Reply:
https://github.com/scality/zenkoclient/blob/feature/ARTESCA-1849/poc/oob.js

Reply:
@miniscruff did you get it to work?

Reply:
Yea I did, just had to clone the zenko client

Reply:
For what it is worth though I did have to edit the script cause I use different names of things